db/migrate/20221020131619_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] The comment table should have an `author_id` column, not a `user_id`. And so these references should be `author`, not `user`.

Reply:
Thanks!